### ISSUE TITLE
How to add folder for Bootstrap app?

### ISSUE THREAD

**Issue by justFredster** [Lundalogik/lip#27]
How do I specify in my package.json that a folder should be copied to the Actionpads\apps folder? Would be great with a general file handling feature. Like: Tell the package.json that content from folder a (path relative to the package.json file) should be moved to folder b (path relative to Actionpads folder.

**Comment by justFredster** [comment]
There is some kind of file copying stuff already, but for LBS apps we want a designated array in the package.json so that these can be selected in the Package Builder.

**Comment by LarsCJensen** [comment]
@justFredster This has been done, yes?

**Comment by justFredster** [comment]
No, not done. Only support for choosing LBS apps in the package builder, but no support for copying the folder to the Actionpads\apps folder when running the LIP installation of the package.
